### PR TITLE
Feature/display-tickets

### PR DESCRIPTION
**Pull request by amarrazza**
## Description

Refactored the dashboard to display ticket information from the resource-tickets endpoints instead of dummy data. Once there is an endpoint for all tickets, the axios request will need to be switched.

@brettsmith212 helped pair program on this pull request.

#### Video Link

https://www.loom.com/share/7596a5e8eeb74967a4e8c83a4196a6e1

#### Trello Link

https://trello.com/c/CPLQkWkp/120-as-a-mentor-i-want-to-be-able-to-see-information-relevant-to-me-as-tickets-so-that-i-can-stay-updated-and-support-my-mentees

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by xokeixo** [comment]
Nice work on displaying the ticket data. Great work.

**Review by clairelist — APPROVED** [review]
Good work on adding this functionality. I like the useEffect pattern here.

File: src/components/pages/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -60,39 +61,38 @@ const columns = [
...
 
 // TODO: make Ant Design Statistics pull ticket totals from ticket tables
 const Dashboard = props => {
+  const [tickets, setTickets] = useState([]);
+
+  useEffect(() => {
+    const getTickets = () => {
+      axiosWithAuth()
+        .get('/resource-tickets')
+        .then(res => {
```
Comment (kimnivore) [inline src/components/pages/Dashboard/Dashboard.js:76]: Great job implementing the axios call to get real data from the backend. Maybe consider adding a .catch for error handling

**Review by kimnivore — APPROVED** [review]

File: src/components/pages/Dashboard/Dashboard.js (hunk 1 above)
Comment (paulstgermain) [reply src/components/pages/Dashboard/Dashboard.js:76]: I would also love to see a `.catch` for error handling here - let's make a note to add it when we circle back to this feature to iron out any kinks.

**Pull request merged**